### PR TITLE
[BZ1980770] Add new IBM Z post install assembly

### PR DESCRIPTION
**Pull request by SNiemann15**
Version(s): 4.8+

Issue: 
https://bugzilla.redhat.com/show_bug.cgi?id=1980770

Link to docs preview:
https://53403--docspreview.netlify.app/openshift-enterprise/latest/post_installation_configuration/ibmz-post-install.html

QE review: Alexander Klein, Holger Wolf
- [x] QE has approved this change.

Additional information: Draft doc: https://docs.google.com/document/d/1OW2ZAhMqt9mEDI0gMGngBa4W6frf0Dh_4yW9O-IMkG4/

### CODE REVIEW

**Comment by SNiemann15** [comment]
For however does the peer-review. I'm not really happy with the callouts <2> for the long strings I have in `modules/ibmz-configure-devices-mco.adoc`. You need to scroll to see callout <2>. But I have no idea how to solve this. Any idea?

File: modules/ibmz-configure-devices-manually.adoc [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+
+.Prerequisites
+
+* You are logged in to the cluster as a user with administrative privileges.
+* The device must be available to the node.
+* In a z/VM environment, the device must be attached to the z/VM guest.
+
+.Procedure
+
+. Connect to the node via ssh:
```
Comment (tmalove) [inline modules/ibmz-configure-devices-manually.adoc:19]:
```suggestion
. Connect to the node via SSH:
```

File: modules/ibmz-configure-devices-manually.adoc [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
++
+[source,terminal]
+----
+$ ssh <user>@<node_ip_address>
+----
++
+Or start a debug session to the node:
```
Comment (tmalove) [inline modules/ibmz-configure-devices-manually.adoc:26]:
```suggestion
You can also start a debug session to the node: 
```

File: modules/ibmz-enable-multipathing-fcp-luns.adoc [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+.Procedure
+
+. Connect to the node via ssh:
++
+[source,terminal]
+----
+$ ssh <user>@<node_ip_address>
+----
++
+Or start a debug session to the node:
```
Comment (tmalove) [inline modules/ibmz-enable-multipathing-fcp-luns.adoc:30]:
```suggestion
You can also start a debug session to the node: 
```

File: modules/ibmz-enable-multipathing-fcp-luns.adoc [hunk 4]
```diff
@@ -0,0 +1,78 @@
...
+====
+
+.Prerequisites
+
+* You are logged in to the cluster as a user with administrative privileges.
+* You have configured multiple paths to a LUN with either method explained above.
+
+.Procedure
+
+. Connect to the node via ssh:
```
Comment (tmalove) [inline modules/ibmz-enable-multipathing-fcp-luns.adoc:23]:
```suggestion
. Connect to the node via SSH:
```

File: modules/ibmz-configure-devices-mco.adoc [hunk 5]
```diff
@@ -0,0 +1,263 @@
...
+     machineconfiguration.openshift.io/role: worker0 <1>
+   name: 99-worker0-devices
+spec:
+   config:
+     ignition:
+       version: 3.2.0
+     storage:
+       files:
+       - contents:
+           source: data:text/plain;base64,QUNUSU9OPT0iYWRkIiwgU1VCU1lTVEVNPT0iY2N3IiwgS0VSTkVMPT0iMC4wLjgwMDAiLCBEUklWRVI9PSJ6ZmNwIiwgR09UTz0iY2ZnX3pmY3BfaG9zdF8wLjAuODAwMCIKQUNUSU9OPT0iYWRkIiwgU1VCU1lTVEVNPT0iZHJpdmVycyIsIEtFUk5FTD09InpmY3AiLCBURVNUPT0iW2Njdy8wLjAuODAwMF0iLCBHT1RPPSJjZmdfemZjcF9ob3N0XzAuMC44MDAwIgpHT1RPPSJlbmRfemZjcF9ob3N0XzAuMC44MDAwIgoKTEFCRUw9ImNmZ196ZmNwX2hvc3RfMC4wLjgwMDAiCkFUVFJ7W2Njdy8wLjAuODAwMF1vbmxpbmV9PSIxIgoKTEFCRUw9ImVuZF96ZmNwX2hvc3RfMC4wLjgwMDAiCg==  <2>
```
Comment (tmalove) [inline modules/ibmz-configure-devices-mco.adoc:79]:
```suggestion
           source: data:text/plain;base64,QUNUSU9OPT0iYWRkIiwgU1VCU1lTVEVNPT0iY2N3IiwgS0VSTkVMPT0iMC4wLjgwMDAiLCBEUklWRVI9PSJ6ZmNwIiwgR09UTz0iY2ZnX3pmY3BfaG9zdF8wLjAuODAwMCIKQUNUSU9OPT0iYWRkIiwgU1VCU1lTVEVNPT0iZHJpdmVycyIsIEtFUk5FTD09InpmY3AiLCBURVNUPT0iW2Njdy8wLjAuODAwMF0iLCBHT1RPPSJjZmdfemZjcF9ob3N0XzAuMC44MDAwIgpHT1RPPSJlbmRfemZjcF9ob3N0XzAuMC44MDAwIgoKTEFCRUw9ImNmZ196ZmNwX2hvc3RfMC4wLjgwMDAiCkFUVFJ7W2Njdy8wLjAuODAwMF1vbmxpbmV9PSIxIgoKTEFCRUw9ImVuZF96ZmNwX2hvc3RfMC4wLjgwMDAiCg==  <2>
```
Here are two suggestions from peers about how to present this callout:
#1 source: data:text/plain;base64,<encoded_base64_string> <2>
#2 source: data:text/plain;base64,QUNUSU9... <2>

Comment (SNiemann15) [reply modules/ibmz-configure-devices-mco.adoc:79]: Thanks good suggestions!

File: modules/ibmz-enable-multipathing-fcp-luns.adoc [hunk 6]
```diff
@@ -0,0 +1,78 @@
...
+. Optional: To format your multipath device with fdisk, run the following command:
++
+[source,terminal]
+----
+$ sudo fdisk /dev/mapper/mpatha
+----
+
+.Verification
+
+. To verify that the devices have been grouped, run the following command:
```
Comment (jeana-redhat) [inline modules/ibmz-enable-multipathing-fcp-luns.adoc:60]: Happened to notice this tiny nit while reviewing for merge, sorry :sweat_smile: 

```suggestion
* To verify that the devices have been grouped, run the following command:
```

**Pull request merged**